src/praisonai-bot/praisonai_bot/gateway/server.py

```
@@ -6102,8 +6102,38 @@ async def start_channels(self, channels_cfg: Dict[str, Dict[str, Any]]) -> None:
+
+        async def _run() -> None:
+            try:
+                await asyncio.sleep(5)
+                await self._replay_outbound_dlq(bot)
+            except asyncio.CancelledError:  # pragma: no cover — shutdown
+                raise
+            except Exception as exc:  # pragma: no cover — recovery is best-effort
+                logger.debug("Outbound crash-recovery task failed: %s", exc)
+
+        try:
+            asyncio.create_task(_run())
```

Ruff (0.16.1)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)